docs/releases/1.17-NOTES.md

```diff
@@ -5,9 +5,15 @@ the notes prior to the release).
 
 # Significant changes
 
-* The default Docker version was changed to 19.03.4. Optional support for Docker 19.03.8 was added and will be the default in future versions. Enable by setting `spec.docker.version: 19.03.8`.
+* The default Docker version has been changed to 19.03.4. Optional support for Docker 19.03.8 has been added and will be the default in future versions. Enable by setting `spec.docker.version: 19.03.8`.
 
-* The default instance type for AWS was changed to t3.medium. This should provide better performance and reduced costs in clusters where the average CPU usage is low.
+* The default instance type for AWS has been changed to t3.medium. This should provide better performance and reduced costs in clusters where the average CPU usage is low.
+
+* Support for [Ubuntu 20.04 (Focal)](../operations/images.md#ubuntu-2004-focal) has been added.
+
```

Comment:
Should mention deprecation of Ubuntu 16.04

Reply:
I added it to 1.18. We are not in a hurry.

Reply:
No reason to defer the announcement.

Reply:
True also. Changed :).